masterplan_tools/specification/schemas/linear_ring.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "linear_ring",
+    "type": "array",
+    "title": "linear ring",
+    "description": "geojson array of positions",
+    "items": {"$ref": "positions"},
```

Comment:
This does not look as a valid JsonSchema as long as it resers to "positions" which is defined in a different file.
I may be wrong as I didn't work with this stuff, but better check. My assumption is that every schema file can be used standalone.